src/main/java/com/amazonaws/services/sqs/AmazonSQSVirtualQueuesClient.java

```diff
@@ -288,7 +288,13 @@ public HostQueue(String queueUrl) {
         }
 
         private void dispatchMessage(Message message) {
```

Comment:
I don't see a unit test for this change. Can we add one and make sure a mock `orphanedMessageHandler` is called if `virtualQueue` or `messageAttributeValue` is null?

Reply:
I added an check that `orphanedMessageHandler ` is called